Explain how to get environment variables in SSR mode (#4029)
withastro/docs: src/content/docs/en/guides/environment-variables.mdx

Because Vite statically replaces `import.meta.env`, you cannot access it with dynamic keys like `import.meta.env[key]`.
:::

> In SSR mode the environment variables can be accessed at runtime based on the SSR adapter. For most adapters you can access environment variables with `process.env`. With the Deno adapter you can get them with `Deno.env.get()`.
> Cloudflare also has its [own way](/en/guides/integrations-guide/cloudflare/#environment-variables) of handling environment variables. Astro will first check the server environment for variables and if they don't exist it will try to look for them in .env files.
> 

Reviewer: ```suggestion
When using SSR, environment variables can be accessed at runtime based on the SSR adapter being used. With most adapters you can access environment variables with `process.env`, for the Deno adapter you have to use `Deno.env.get()`. Cloudflare has its [own way](/en/guides/integrations-guide/cloudflare/#environment-variables) of handling environment variables. 
Astro will check the server environment for variables, if they don't exist, Astro will look for them in .env files.

```

Author: Thank you for the suggestions! 🙌
